notebooks/wav2lip/wav2lip.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    pip_install("-q", "openvino&gt;=2024.3.0")
</pre><p>2024.4.0,</p><p> </p><p>also not sure how contrib and general opencv packages interact with each other (if I right remember contrib includes base and override content, but I may mistaken)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>openvino version is updated. I checked opencv packages, they are installing independently.</p>